f8a_worker/dispatcher/flows/bayesianAnalysisFlow.yml

```diff
@@ -8,7 +8,7 @@
        - 'bayesianFlow'
        - 'bayesianApiFlow'
        - 'bayesianPriorityFlow'
-       - 'bayesianPackageFlow'
+       # - 'bayesianPackageFlow'
```

Comment:
Ah, I see now. I see why the no-such-bucket error disappeared. It's because of this change. We will wait for package level data to land in S3/minio before we call GraphImporter task. And we call the task twice, once for component-level data and second time for package-level data.

To be honest, this feels like a workaround. With this change we will keep the flow control message on `bayesianAnalysisFlow` queue much longer than actually needed. Dispatcher workers will need to periodically check the state of the flow. This could slow the system down as we talk about thousands (even 50k-100k sometimes) of messages which will consume some of our precious resources.

@tuxdna let's talk this through :wink:

Reply:
Update the discussion here: https://github.com/openshiftio/openshift.io/issues/2219#issuecomment-375276749